{{cookiecutter.project_slug}}/clients/web/react/src/services/axios-instance.ts

```diff
@@ -1,8 +1,11 @@
 import axios, { AxiosError } from 'axios'
 import { localStoreManager } from 'src/utils/local-store-manager'
 import { getCookie } from 'src/utils/get-cookie'
-const axiosInstance = axios.create({
-  //baseUrl will be determined by the server proxy in vite.config.js
+
+const baseURL = `${window.location.protocol}//${window.location.host}`
```

Comment:
How does this interact with vite's proxy?

Reply:
it replaces the current window location if it is in dev and rewrites the host, I think it is more helpful to have it be explicit rather than assumed